src/emmetHelper.ts

```diff
@@ -169,6 +171,10 @@ function getCurrentWord(document: TextDocument, position: Position): string {
 	}
 }
 
+function makeCursorsGorgeous(expandedWord: string): string {
+	return expandedWord.replace(/\$\{\d+\}/g, '|').replace(/\$\{\d+:([^\}]+)\}/g, '_$1_');
```

Comment:
about the tests, I wanted to get your feature in last night's Insiders, so couldn't wait :)

I don't see it in cursive characters. Is that a markdown syntax? If yes, then the suggestion docs don't support markdown (yet). So the `_` were showing up as just `_`